src/encoded/static/components/search.js

```diff
@@ -346,8 +346,8 @@ const ExperimentComponent = (props, reactContext) => {
     }
 
     // Get SCREEN link and FactorBook link if they exist
```

Comment:
Are you trying to do this:
const screenLink  = result.dbxrefs  && result.dbxrefs.find( dbxref => (dbxref.indexOf('SCREEN') > -1)  )

If not, you can use "some" to check for any dbxref and find to get the first instance, like:
const screenLink = (result.dbxrefs && result.dbxrefs.**some**(dbxref => (dbxref.indexOf('SCREEN') > -1))) ? result.dbxrefs.**find**(dbxref => (dbxref.indexOf('SCREEN')))[0] : null;

I did not test it out so it may be off by a parenthesis or two.

Reply:
I changed this. I always forget about `.some`, somehow.